plans/main.fmf

```diff
@@ -3,21 +3,18 @@ discover:
 execute:
     how: tmt
 prepare:
-    how: feature
-    epel: enabled
-
-/upstream:
-    summary: Run SELinux specific Podman tests on upstream PRs
-    discover+:
-        filter: tag:upstream
-    adjust+:
-        enabled: false
-        when: initiator is not defined or initiator != packit
-
-/downstream:
-    summary: Run SELinux specific Podman tests on bodhi / errata and dist-git PRs
-    discover+:
-        filter: tag:downstream
-    adjust+:
-        enabled: false
-        when: initiator == packit
+    - when: distro == centos-stream or distro == rhel
+      how: shell
+      script: |
+        dnf -y install https://dl.fedoraproject.org/pub/epel/epel-release-latest-$(rpm --eval '%{?rhel}').noarch.rpm
+        dnf -y config-manager --set-enabled epel
+      order: 10
+    - when: initiator == packit
+      how: shell
+      script: |
+        COPR_REPO_FILE="/etc/yum.repos.d/*podman-next*.repo"
+        if compgen -G $COPR_REPO_FILE > /dev/null; then
+            sed -i -n '/^priority=/!p;$apriority=1' $COPR_REPO_FILE
+        fi
```

Comment:
would something like this work? I think it's more readable

`dnf config-manager --save --setopt="podman-next*.priority=1"`

Reply:
actually I just pulled it out from memory and I am not sure if it's going to be the same for dnf4/dnf5. I should verify that

Reply:
Things have changed quite a bit between dnf4 and dnf5. So, unless we have something that works across the board, I'd prefer to keep it this way. I pulled this from packit project's own TMT config btw.

Reply:
right, this actually changes from dnf4/dnf5. I just verified that and in dnf5 it would be

`dnf config-manager setopt "copr:copr.fedorainfracloud.org:rhcontainerbot:podman-next.priority=1"`

which would add few lines in 

```
/etc/dnf/repos.override.d/99-config_manager.repo

[copr:copr.fedorainfracloud.org:rhcontainerbot:podman-next]
priority=1
```

different command, different file: I am good with `sed`

Reply:
so LGTM!